### DISCUSSION TITLE
使用onedrive作为存储策略，最后一步授权的的时候一直报错 Failed to fetch AccessToken

### DISCUSSION THREAD

**Discussion by giszh86** [cloudreve/cloudreve#3269] [General]
使用onedrive作为存储策略，最后一步授权的的时候一直报错Failed to fetch AccessToken

**Comment by ghost** [comment]
添加第一个pointshare没问题，第二个就是报错 ，错误Failed to fetch AccessToken,

**Comment by Ecss11** [comment]
检查应用创建的密钥（Client Secrets）是否正确，**需要复制值（Value）而不是 Secret ID**，复制 Secret ID 是没有用的。

**Comment by wangyijin209** [comment]
> 检查应用创建的密钥（Client Secrets）是否正确，**需要复制值（Value）而不是 Secret ID**，复制 Secret ID 是没有用的。

复制的是值依旧报错，使用docker搭建

**Comment by Ecss11** [comment]
> > 检查应用创建的密钥（Client Secrets）是否正确，**需要复制值（Value）而不是 Secret ID**，复制 Secret ID 是没有用的。
> 
> 复制的是值依旧报错，使用docker搭建

我使用的也是 docker，你能导出日志或者提供一个具体的错误提示吗？

**Comment by wangyijin209** [comment]
> > > 检查应用创建的密钥（Client Secrets）是否正确，**需要复制值（Value）而不是 Secret ID**，复制 Secret ID 是没有用的。
> > 
> > 
> > 复制的是值依旧报错，使用docker搭建
> 
> 我使用的也是 docker，你能导出日志或者提供一个具体的错误提示吗？

我是CDN问题，先暂时关闭CDN或回源就可以了

**Comment by Chang-Tao** [comment]
有两个关键参数， 第一步是复制应用程序ID， 直接在概览页面复制
第二部是证书密码，创建证书密码是生成两个参数，一个是值，一个是机密ID， 只会在第一次创建时显示，离开这个页面就不会显示， 复制那个值，而不是机密ID，就可以成功。

**Comment by tourusa** [comment]
我也遇到同样的问题了，也没有开 CDN，还有什么解决办法啊？

**Comment by StarYuhen** [comment]
> 我也遇到同样的问题了，也没有开CDN，还有什么解决办法啊？

尝试登陆https://portal.azure.com/，查看自己cloudreve的账号所绑定应用程序的凭证是否过期，我重新建了个凭证就修复了
